extensions/objects/models/objects.py

```diff
@@ -245,6 +245,7 @@ def get_schema(cls):
         }
 
 
+# TODO(shavavo): Migrate SubtitledUnicode to TranslatableUnicode.
```

Comment:
Please always use issue numbers in TODOs, ditto below.

Reply:
Done